ui/src/testing/react-test-helpers.ts

```diff
@@ -8,10 +8,12 @@ import {act} from 'react-dom/test-utils';
 // And fails with:
 //   ReferenceError: Zone is not defined
 export async function waitOneTickAndUpdate(wrapper: ReactWrapper) {
-  await act(() => new Promise(setImmediate).then(() => wrapper.update()));
+  // @ts-ignore
```

Comment:
why ts-ignore these? Please add a comment

Note: I did experience similar issues when trying to upgrade Jest a while back. Details are on https://precisionmedicineinitiative.atlassian.net/browse/RW-5959